smtk/attribute/System.cxx

```diff
@@ -905,3 +905,32 @@ smtk::common::ViewPtr System::findViewByType(const std::string &vtype) const
   return smtk::common::ViewPtr();
 }
 //----------------------------------------------------------------------------
+smtk::common::ViewPtr System::findTopLevelView() const
```

Comment:
The other methods have findXX and findXXs - I think this pattern is ok (unless you want to change the other methods as well).